### PR TITLE
Make small changes to the documentation

### PR DESCRIPTION
**Pull request by qubixes**
I have thoroughly reviewed the documentation on readthedocs. I have made a few small changes.

### CODE REVIEW

File: docs/source/about/metasyn_in_detail.rst [hunk 1]
```diff
@@ -101,52 +102,58 @@ Model estimation starts with an appropriately pre-processed data frame. For ``me
...
-     - UniformDistribution, NormalDistribution, ...
+     - UniformDistribution
    * - discrete
      - int
      - 1, 2, ...
      - DiscreteUniformDistribution
    * - categorical
      - pl.Categorical
-     - gender, country
+     - Male, Female, Male, Female
```
Comment (Samuwhale) [inline docs/source/about/metasyn_in_detail.rst:126]:
```suggestion
     - Male, Female, Non-binary, Female
```

Tiny inclusivity change

File: docs/source/usage/generating_metaframes.rst [hunk 2]
```diff
@@ -69,7 +69,7 @@ spec
         
         It is safe to ignore this warning - however, be aware that without setting the column as unique, metasyn may generate duplicate values for that column when synthesizing data.
         
-        To remove the warning and ensure the column remains unique, set the column to be unique (``"column" = {"unique": True}``) in the ``spec`` dictionary.    
+        To remove the warning and ensure the values in the synthetic column are unique, set the column to be unique (``"column" = {"unique": True}``) in the ``spec`` dictionary.
```
Comment (Samuwhale) [inline docs/source/usage/generating_metaframes.rst:72]:
```suggestion
        To remove the warning and ensure the values in the synthesized column are unique, set the column to be unique (``"column" = {"unique": True}``) in the ``spec`` dictionary.    
```

I like synthesized here more as it implies being a result of being generated by metasyn, as opposed to simply being fake data.

**Review by Samuwhale — COMMENTED** [review]
Looks good, some very nice changes :)

File: docs/source/about/metasyn_in_detail.rst (hunk 1 above)
Comment (qubixes) [reply docs/source/about/metasyn_in_detail.rst]: Thanks! I have changed it to Yes/No/Maybe. That should suffice as well I think.

**Review by Samuwhale — APPROVED** [review]

**Pull request merged**